modAionImpl/src/org/aion/zero/impl/db/AionRepositoryCache.java

```diff
@@ -624,4 +638,25 @@ public boolean isIndexed(byte[] hash, long level) {
     public byte getVMUsed(Address contract) {
         return repository.getVMUsed(contract);
     }
+
+    @Override
+    public void setTransformedCode(Address contractAddr, byte[] code) {
```

Comment:
Requires write locks.

Reply:
sure. will add it.

Reply:
https://github.com/aionnetwork/aion/pull/870/commits/cd6b10d512065c88293c4befad3dcb3d0fe56307